
syntax = "proto3";

enum TextType {
    NEWS = 0;
    OPINION = 1;
    ADVICE = 2;
}

message TextTypeRequest {
    string raw_text = 1;
}

message TextTypeResponse {
    TextType type = 1;
}

service Types {
    rpc GetType (TextTypeRequest) returns (TextTypeResponse);
}